chainer/functions/basic_math.py

```diff
@@ -40,6 +46,18 @@ class AddConstant(function.Function):
     def __init__(self, value):
         self.value = value
 
+    def label(self):
+        value = self.value
+        if isinstance(value, float) or\
+           isinstance(value, numpy.ndarray) or\
+           isinstance(value, cuda.GPUArray):
+            return "+ %s" % str(value)
```

Comment:
If the value is an array, the string may be very long. How about just printing `'constant array'` if the value is an array?